elements/dash-checkbox/dash-checkbox.js

```diff
@@ -103,6 +103,11 @@
      */
     set disabled(isDisabled) {
       this.setAttribute('aria-disabled', isDisabled);
+      if (isDisabled) {
```

Comment:
I'm a little sleepy but I _think_ we don't need to because it'll get triggered by `attributeChangedCallback` when the element boots if the user has added `[disabled]`. If the user has NOT added disabled, then what's in connectedCallback will happen first and set tabindex to 0